### DISCUSSION TITLE
What macOS versions does the plugin support?

### DISCUSSION THREAD

**Discussion by churnish** [bandantonio/obsidian-apple-books-highlights-plugin#49] [Q&A]
Hi! I wanted to file a bug report because I couldn't import any of my highlights on macOS Big Sure but I noticed that there wasn't an option for Big Sur in the macOS version section. Does this mean that this plugin only supports Monterey and up? 😭

FWIW, I've successfully located all my highlights at `~/Library/Containers/com.apple.iBooksX/Data/Documents/AEAnnotation/`

**Comment by bandantonio** [comment]
@greetclammy Thank you for the question.
Although the oldest macOS version the plugin has been tested on was Monterrey, there shouldn't be any restrictions on using it on BigSur. Have you checked the console? Are there any errors?

**Reply by churnish** [reply]
Right! The console says:
>plugin:apple-books-i…ort-highlights:6757 Uncaught (in promise) No books found. Looks like your Apple Books library is empty.

It's certainly not empty!
